src/theme/dark/colors.js

```diff
@@ -12,6 +12,7 @@ const appColors = {
   dropdown: rawColors.neutral.outerSpace,
   elementBackground: rawColors.neutral.outerSpace,
   elementBackgroundHover: rawColors.neutral.tuna,
+  headerItemBackground: rawColors.neutral.limedSpruce,
```

Comment:
The naming is off here, maybe `importantNoticeBackground`? or something more specific to what the component is.